src/components/PopUpForm/PopUpForm.tsx

```diff
@@ -0,0 +1,274 @@
+import {
+  Form,
+  FormLayout,
+  TextField,
+  Button,
+  Card,
+  ActionList,
+  Popover,
+  Stack,
+  TextStyle,
+  Banner,
+} from "@shopify/polaris";
+import React, { useState } from "react";
+import { postalCodeIsValid } from "../../utils";
+
+export function PopUpForm() {
+  const [shouldShowInvalidName, setShouldShowInvalidName] = useState(false);
+  const [shouldShowInvalidDate, setShouldShowInvalidDate] = useState(false);
+  const [shouldShowInvalidAddress, setShouldShowInvalidAddress] = useState(
+    false,
+  );
+  const [shouldShowInvalidCity, setShouldShowInvalidCity] = useState(false);
+  const [shouldShowInvalidProvince, setShouldShowInvalidProvince] = useState(
+    false,
+  );
+  const [shouldShowInvalidPostal, setShouldShowInvalidPostal] = useState(false);
+  const [shouldShowInvalidURL, setShouldShowInvalidURL] = useState(false);
```

Comment:
Do we need something to track if the phone is in the wrong format?

Reply:
We can, I just assumed since only mods will be entering data, no need to sanitize it

Reply:
Well, not sanitize but make sure its the correct number of digits - prevent people from fat fingering the value